owncast_for_windows.md

```diff
@@ -0,0 +1,139 @@
+<br />
+<p align="center">
+  <a href="https://github.com/owncast/owncast" alt="Owncast">
+    <img src="https://owncast.online/images/logo.png" alt="Logo" width="200">
+  </a>
+</p>
+
+<br/>
+
+  <p align="center">
+    <strong>Take control over your content and stream it yourself.</strong>
+    <br />
+    <a href="https://owncast.online"><strong>Explore the docs »</strong></a>
+    <br />
+    <a href="https://watch.owncast.online/">View Demo</a>
+    ·
+    <a href="https://broadcast.owncast.online/">Use Our Server for Testing</a>
+    ·
+    <a href="https://owncast.online/faq/">FAQ</a>
+    ·
+    <a href="https://github.com/owncast/owncast/issues">Report Bug</a>
+  </p>
+</p>
+
+# Owncast-for-windows
+
+> Note:  Owncast currently **does not support Windows Operating System**. This is a workaround to run Owncast on Windows.
+
+
+This document list out the steps in detail to install and run Owncast in Windows using Windows Subsystem for Linux, specifically **WSL2**.  
+The steps provided are for both the use cases whether you want to use Owncast or build and install Owncast for contributing to the project.
+
+---
+
+### Installing WSL2 in Windows.
+There are lots of tutorials available online (videos and docs both) on how to install WSL2.  
+Here are the official documents from Microsoft -> [Install Linux on Windows with WSL](https://learn.microsoft.com/en-us/windows/wsl/setup/environment)
+Some points to remember -> 
+- Preferable method to install WSL2 is by using the `wsl --install `. If you are facing issues with this method you can look at - [Manual installation steps for older versions of WSL](https://learn.microsoft.com/en-us/windows/wsl/install-manual)
+- Make sure you have enabled the Virtual Machine feature. (ignore if used wsl --install method)
+- Make sure you have WSL2
+- Installed your Linux distribution of choice and make sure you installed the latest available version (Preferably Ubuntu)
+
+
+### Setting up WSL2 and the distribution of your choice.
+After basic setup, you can look into setting WSL2 for development. Here is the link for a detailed document by Microsoft - [https://learn.microsoft.com/en-us/windows/wsl/setup/environment](https://learn.microsoft.com/en-us/windows/wsl/setup/environment)
+
+### Make sure all the prerequisites are installed in WSL2.
+Here is the list for all the prerequisites required ->
+ - C compiler, such as [GCC compiler](https://gcc.gnu.org/install/download.html) or a [Musl-compatible compiler](https://musl.libc.org/)
+ - npm (Node Package Manager) is installed as `sudo apt install npm`. 
+ - Node.js is installed (LTS Version) `sudo apt install nodejs`.
+ - [ffmpeg](https://ffmpeg.org/download.html)
+ - Install the [Go toolchain](https://golang.org/dl/) (1.18 or above).
+
+<hr>
+
+## Now for users who just want to use Owncast
```

Comment:
Since more people want to simply run Owncast than to develop Owncast the "use" should probably be before the "develop" part, because most people don't need to know about compilers, nodejs, etc.